@YichenZW @haojinw0027 This PR doesn't seem to support the default example in `examples/api_collm.json`, getting the following error:

```
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/gpfs/projects/argon/shangbin/model_collaboration/model_collaboration/main.py", line 80, in <module>
    run_main()
  File "/gpfs/projects/argon/shangbin/model_collaboration/model_collaboration/main.py", line 55, in run_main
    result = method_module.run_method(
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/gpfs/projects/argon/shangbin/model_collaboration/model_collaboration/method/api_collm.py", line 946, in run_method
    generator_scorer.run()
  File "/gpfs/projects/argon/shangbin/model_collaboration/model_collaboration/method/api_collm.py", line 281, in run
    scored_data = self.scorer.process(
                  ^^^^^^^^^^^^^^^^^^^^
  File "/gpfs/projects/argon/shangbin/model_collaboration/model_collaboration/utils/collm_scoring.py", line 732, in process
    data_dict = self.encode_dataset(
                ^^^^^^^^^^^^^^^^^^^^
  File "/gpfs/projects/argon/shangbin/model_collaboration/model_collaboration/utils/collm_scoring.py", line 593, in encode_dataset
    assert raw_datasets[0]["messages"][0]["role"] == "user", "When use_completion_format is True, the first message must be user message to ensure correct prompt-completion split."
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: When use_completion_format is True, the first message must be user message to ensure correct prompt-completion split.
```

Please see why and edit the PR to fix this?

--- a/model_collaboration/method/api_collm.py
+++ b/model_collaboration/method/api_collm.py
@@ -939,7 +939,8 @@ def run_method(task, task_type, gpu_ids, model_names, hyperparameters):
             output_dir=f"{collm_dir}/generator_scored_data",
             max_seq_length=max_sequence_length,
             use_flash_attn=False,
-            use_completion_format=True,
+            use_completion_format=False,
+            preprocessing_format="qwen_chat",
             use_slow_tokenizer=True,
             preprocessing_num_workers=16
         )
@@ -954,7 +955,8 @@ def run_method(task, task_type, gpu_ids, model_names, hyperparameters):
             output_dir=f"{collm_dir}/mentor_scored_data",
             max_seq_length=max_sequence_length,
             use_flash_attn=False,
-            use_completion_format=True,
+            use_completion_format=False,
+            preprocessing_format="qwen_chat",
             use_slow_tokenizer=True,
             preprocessing_num_workers=16
         )

